### PR TITLE
Added warning about 222 blocked and fixed typo

### PR DESCRIPTION
**Pull request by slashzero**
This came up in a ticket and was confusing.

### CODE REVIEW

**Comment by nataliejeremy** [comment]
Thanks! @slashzero

File: source/docs/articles/local/ssh-tunnels-for-secure-connections-to-pantheon-services.md [hunk 1]
```diff
@@ -30,6 +30,10 @@ Use the required values from the **Connection Info** tab, the desired environmen
 ```
 ssh -f -N -L PORT:localhost:PORT -p 2222 ENV.SITE_UUID@dbserver.ENV.SITE_UUID.drush.in
 ```
+PORT is the database port specified in the **Connection Info** tab.
+<div class="alert alert-info" role="alert">
+<h4>Note</h4>
```
Comment (rachelwhitton) [inline source/docs/articles/local/ssh-tunnels-for-secure-connections-to-pantheon-services.md]: Suggest moving this note to the troubleshooting section below or to include it within the above statement - to reduce clutter of notes. @nataliejeremy what do you think?

**Comment by rachelwhitton** [comment]
Thanks for the PR @slashzero!

File: source/docs/articles/local/ssh-tunnels-for-secure-connections-to-pantheon-services.md (hunk 1 above)
Comment (nataliejeremy) [inline source/docs/articles/local/ssh-tunnels-for-secure-connections-to-pantheon-services.md]: +1 for the troubleshooting section

**Comment by rachelwhitton** [comment]
+1

**Pull request merged**